### ISSUE TITLE
new Minus

### ISSUE THREAD

**Issue by DanGrayson** [Macaulay2/M2#3246]
```
i6 : new Minus from {a,b,c}

o6 = -a

o6 : Expression of class Minus
```

An error message should be issued here, because the initializing list has length greater than 1.

**Comment by d-torrance** [comment]
Or should `net(Minus)` be updated to return `-{a,b,c}` instead?

**Comment by pzinn** [comment]
No,`net` works correctly, a `Minus` is supposed to be of length 1.
That being said I don't understand the original issue. There are plenty of "illegal" objects one can create using the "new" construction. The latter is low level and therefore allows (and should allow) to construct essentially anything.
For a given class, one can always create a high level constructor (e.g., `hashTable` for `HashTable`) that can do all kinds of checks as needed.

**Comment by mahrud** [comment]
```m2
i1 : new GradedModule from { blah => foo }

o1 = 0

o1 : GradedModule

i2 : peek oo

o2 = GradedModule{blah => foo}
```

**Comment by d-torrance** [comment]
I'm going to close this -- Paul made a convincing argument that this isn't really a bug
